src/jit/morph.cpp

```diff
@@ -2995,8 +3000,6 @@ GenTreeCall* Compiler::fgMorphArgs(GenTreeCall* callNode)
         {
```

Comment:
We could probably also fix this #ifdef as well, like this:

else
#endif // !UNIX_AMD64_ABI
{
   passUsingFloatRegs = varTypeIsFloating(argx); 
}